shell.nix

```diff
@@ -5,6 +5,6 @@ in
   haskell.lib.buildStackProject {
      ghc = haskell.packages.${lts}.ghc;
      name = "rscoin";
-     buildInputs = [ zlib glib cairo gnome.pango gnome3.gtk git cabal-install
+     buildInputs = [ zlib.out zlib.dev glib cairo gnome.pango gnome3.gtk git cabal-install
```

Comment:
I pushed a fix, please test locally, it does work for me using release-16.09 nixpkgs branch.